tests/baselines/reference/objectTypeLiteralSyntax2.errors.txt

```diff
@@ -15,4 +17,8 @@ tests/cases/conformance/types/objectTypeLiteral/objectTypeLiteralSyntax2.ts(12,2
     
     var z: { foo: string bar: string }
                          ~~~
+!!! error TS1005: '=' expected.
+                         ~~~
+!!! error TS2304: Cannot find name 'bar'.
+                            ~

```

Comment:
I'm not sure if we should accept these errors, as the ```'=' expected``` message implies that initializers are allowed in this case. Maybe we should spin this off into a new issue. @DanielRosenwasser @CyrusNajmabadi, what are your thoughts on this?